### DISCUSSION TITLE
Book Spy! The command to see what people write in books, signed or unsigned!

### DISCUSSION THREAD

**Discussion by AutumnUwU** [Zrips/CMI#7726] [Feature Requests]
/bookspy
cmi.command.bookspy
Idea behind it is being able to see what anyone writes in a book regardless signed or unsigned. Such as what this plugin does:
https://www.spigotmc.org/resources/jspy.14968/

Except also feature ability to change the message it sends as well as make it hex compatible

I'd love to see this feature in the future pleaseee! Would be so nice instead of having to add more plugins.  

Edit: Maybe /anvilspy too to see what people rename stuff on anvils

**Comment by CloudeLecaw** [comment]
Pls move it to discussions :)
